docker-compose/Makefile

```diff
@@ -14,12 +20,13 @@ WSK_CLI ?= $(PROJECT_HOME)/bin/wsk
 OPEN_WHISK_DB_PREFIX ?= local_
 
 DOCKER_KERNEL ?= $(shell docker version --format "{{.Server.KernelVersion}}")
-RUNC_BINARY   ?= $(shell if [[ $(DOCKER_KERNEL) == *-moby || $(DOCKER_KERNEL) ==  *-boot2docker ]]; then (docker run --rm --privileged --pid=host debian nsenter -t 1 -m -u -n -i sh -c "which runc || which docker-runc"); else (which runc || which docker-runc); fi)
-DOCKER_BINARY ?= $(shell if [[ $(DOCKER_KERNEL) == *-moby || $(DOCKER_KERNEL) ==  *-boot2docker ]]; then (docker run --rm --privileged --pid=host debian nsenter -t 1 -m -u -n -i sh -c "which docker"); else (which docker); fi)
-
-
-UNAME_STR ?= $(shell uname)
-
+ifeq ("$(UNAME_STR)","Linux")
+	RUNC_BINARY   ?= $(shell (which runc || which docker-runc))
```

Comment:
Was this command failing  in `Linux` ? I think this happens anyway when the `DOCKER_KERNEL` is not `*-moby` or `*_boot2docker`.

Reply:
So I think the default shell my make is using is bourne not bash. so the '[[' blows up straight out of the gate. I basically end around all the other stuff cause you can and should just run docker natively on linux. You can see travis failing with the same errors I was:

```bash
/bin/sh: 1: [[: not found
/bin/sh: 1: 4.4.0-101-generic: not found
/bin/sh: 1: [[: not found
/bin/sh: 1: 4.4.0-101-generic: not found
```

Reply:
got it. thanks for the details @mcdan